File: print-manager-dotnet-core-cs/SimplePrint.cs
using System;
using ceTe.DynamicPDF.Printing;

namespace print_manager_dotnet_core_cs
{
    class SimplePrint
    {
        public static void Run()
        {
            // Create a print job with the document to be printed to the default printer
            PrintJob printJob = new PrintJob(Printer.Default, Util.GetPath("Resources/DocumentA.pdf"));

            // Print the print job
            printJob.Print();
        }
    }
}

File: print-manager-dotnet-core-vb/SimplePrint.vb
Imports ceTe.DynamicPDF.Printing

Public Class SimplePrint
    Shared Sub Run()
        ' Create a print job with the document to be printed to the default printer
        Dim MyPrintJob As PrintJob = New PrintJob(Printer.Default, Util.GetPath("Resources/DocumentA.pdf"))

        ' Print the print job
        MyPrintJob.Print()
    End Sub
End Class
